src/goto-symex/goto_symex_state.cpp

```diff
@@ -798,23 +798,13 @@ ssa_exprt goto_symex_statet::add_object(
 
   ssa_exprt ssa = rename_ssa<L0>(ssa_exprt{expr}, ns).get();
   const irep_idt l0_name = ssa.get_identifier();
-  const std::size_t l1_index = index_generator(l0_name);
+  const auto l1_index = narrow_cast<unsigned>(index_generator(l0_name));
```

Comment:
Why `unsigned` and not `size_t`?

Reply:
`symex_renaming_levelt` uses `unsigned` for indexes